backend/data.go

```diff
@@ -14,18 +13,14 @@ type QueryDataHandler interface {
 	// req contains the queries []DataQuery (where each query contains RefID as a unique identifer).
 	// The QueryDataResponse contains a map of RefID to the response for each query, and each response
 	// contains Frames ([]*Frame).
-	QueryData(ctx context.Context, req *QueryDataRequest) (*QueryDataResponse, error)
+	QueryData(pCtx PluginContext, req *QueryDataRequest) (*QueryDataResponse, error)
```

Comment:
Talked about this in call, should be two arguments, go context and then plugincontext, or something other name (but def two objects). Overloading go's context can result in problems based on past experience.

Reply:
I fixed that